### DISCUSSION TITLE
Do we still need that Makefile

### DISCUSSION THREAD

**Discussion by MatrixCrawler** [warrensbox/terraform-switcher#385] [Ideas]
Is this in use somewhere since we switched to github-actions for the build process?

**Comment by warrensbox** [comment]
Actually I do use the Makefile for local development (and I still use it)

**Reply by yermulnik** [reply]
> Actually I do use the Makefile for local development (and I still use it)

I support that (despite I don't use is as I don't do lots of development on `tfswitch` often).
